### PR TITLE
Changes the reference player in KogaPlanetLauncher,java from static t…

### PR DESCRIPTION
**Pull request by YuniCodeArt**
In KogaPlanetLauncher.java I changed the line 29 to "private Player player", then I changed the rest of the code in KogaPlanetLauncher.java instead of using "PLAYER" to use "player" as reference.

Still in the KogaPlanetLauncher.java, there's a reference called "gui" that instances "PlayerGui" in the line 84, and in that in the second parameter I changed "PLAYER" to "player" to make the player reference go inside to the PlayerGui, then to use the player reference in the instantiation of "menuGui", that's why in "MenuGui.java" I added the new parameter for the constructor called "Player player" (I created the reference for the player inside "MenuGui.java" for getting that player reference from the constructor and using it in the line 67 in "MenuGui.java", to use the method ".isAlive" from the player).

### CODE REVIEW

File: desktop/.project [hunk 1]
```diff
@@ -24,7 +24,7 @@
 		<link>
 			<name>assets</name>
 			<type>2</type>
-			<location>/home/finalshare/Documentos/1-Programacao/Arquivos java/KGP/Area de trabalho/kogaplanet-launcher/assets</location>
```
Comment (Finalshare90) [inline desktop/.project:27]: Ops, little mistake here, i should to add that little guy to the gitignore file.

File: desktop/bin/main/.gitignore
Comment (Finalshare90) [inline desktop/bin/main/.gitignore:1]: Is that referring only to the Bin asset files, or to the entire asset folder?

**Review by YuniCodeArt — COMMENTED** [review]
Apparently everything right.

**Review by Finalshare90 — APPROVED** [review]
Ok! everything looks nice:)

Did some changes with .gitignore on your project, and adjusted others things.

_Thanks for your support!_

**Pull request merged**